include/Plugin.h

```diff
@@ -101,12 +147,28 @@ class LMMS_EXPORT Plugin : public Model, public JournallingObject
 
 				inline bool isValid() const
 				{
-					return desc != NULL && name.isNull() == false;
+					return desc != nullptr;
```

Comment:
Is this change needed for this pr?

Reply:
The `name` class members can be nullptr while the whole key is valid. `name` is only used in some situations and must be considered temporary. See my comment above the `Key::name` member.

Also, the change should be harmless to old code, as it has only ever be used inside `EffectSelectDialog::acceptSelection`.